gulpfile.js

```diff
@@ -137,4 +139,8 @@ gulp.task('patch', function() {
     runSequence('bump-patch', 'build', 'update-version');
 });
 
-gulp.task('build', ['css', 'js', 'css-min', 'js-min']);
+gulp.task('build', function() {
+    runSequence('lint', 'css', 'js', 'css-min', 'js-min', 'update-version')
```

Comment:
How can you say they depend on each other when I see they don't? I've worked with Gulp long enough. Show me the exact lines.